src/main/java/com/groupseven/sysc4806project/Book.java

```diff
@@ -0,0 +1,85 @@
+package com.groupseven.sysc4806project;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+
+import java.awt.*;
+import java.io.Serializable;
+@Entity
+public class Book implements Serializable {
+    private int id;
+    private Image picture;
```

Comment:
I have an alternate idea for images. When adding a new Book we could accept an image as a parameter and save it under `resources/static/{book_id}.png` with a default empty image returned if no image exists. Could eliminate the need to have this field and the `import java.awt.*`. What do you think?

Reply:
Just to add, when we display the image in HTML it will be through an `<img>` tag that takes a URL to an image. This would facilitate that

Reply:
Yea, i think that would work

Reply:
We could still have a `getPicture()`, but that would just return a String with a link to the image on the server, not the image itself. If we define some sort of convention for images (ie: only save them under `resources/static/{book_id}.png`), we could generate this URL from within the class with just the book ID

Reply:
ah I see what you mean, then I agree as well sounds good!